lms/static/js/student_account/views/FormView.js

```diff
@@ -185,32 +176,32 @@
 
                 saveError: function(error) {
                     this.errors = ['<li>' + error.responseText + '</li>'];
-                    this.setErrors();
+                    this.renderErrors(this.defaultFormErrorsTitle, this.errors);
                     this.toggleDisableButton(false);
                 },
 
-                setErrors: function() {
-                    var $msg = this.$errors.find('.message-copy'),
-                        html = [],
-                        errors = this.errors,
-                        i,
-                        len = errors.length;
-
-                    for (i = 0; i < len; i++) {
-                        html.push(errors[i]);
-                    }
+                renderErrors: function(title, errorMessages) {
+                    this.renderFormFeedback(this.formErrorsTpl, {
+                        title: title,
+                        messagesHtml: HtmlUtils.HTML(errorMessages.join(''))
+                    });
+                },
 
-                    $msg.html(html.join(''));
+                renderFormFeedback: function(template, context) {
+                    this.clearFormFeedback();
 
-                    this.element.show(this.$errors);
+                    HtmlUtils.append(
+                        this.$formFeedback,
+                        HtmlUtils.template(template)(context)
```

Comment:
To make the code more easily digestible I think it would be good to assign the template to a variable, then pass the context to said variable. e.g.
```
var tpl = HtmlUtils.template(template);
tpl(context);
```